oggm/core/inversion.py

```diff
@@ -151,7 +151,29 @@ def shape_factor_huss(widths, heights, is_rectangular):
     -------
     shape factor (no units)
     """
-    return widths / (2 * heights + widths)
+    # Ensure bool (for masking)
+    is_rectangular = is_rectangular.astype(bool)
+
+    shape_factors = np.ones(widths.shape)
+    shape_factors[~is_rectangular] = \
+        (widths / (2 * heights + widths))[~is_rectangular]
+
+    return shape_factors
+
+
+# TODO: how to handle zeta > 10? at the moment extrapolation
+# Table 1 from Adhikari (2012) and corresponding interpolation functions
+_ADHIKARI_TABLE_ZETAS = np.array([0.5, 1, 2, 3, 4, 5, 10])
```

Comment:
Move this to top of file?

Reply:
I'm not sure there is a pep rule for this, but yes I think it's slightly better